client/src/Settings.js

```diff
@@ -1,6 +1,6 @@
 let Settings = {
     currentPort: "http://localhost:3000",
-    githubClientId: "d8ca7de576a6e29f75ca",
+    githubClientId: "cfa78f27867826309fb6",
```

Comment:
This should be parameterized also.

Reply:
These are something that should probably be read out of the `.env` file for local development. In production these would be set as environment variables at runtime. Also make sure the documentation is updated to add these parameters.

Reply:
updated the file to read from the env file in new commit so resolving this